va/caa_test.go

```diff
@@ -694,93 +706,93 @@ func TestMultiCAARechecking(t *testing.T) {
 			expectedProbSubstring: "While processing CAA for present-dns-only.com: dnsClient is broken",
 			expectedProbType:      probs.DNSProblem,
 			remoteVAs: []RemoteVA{
-				{remoteVA, remoteUA},
-				{remoteVA, remoteUA},
-				{remoteVA, remoteUA},
+				{remoteVA, testPerspective1, testRIR1},
+				{remoteVA, testPerspective2, testRIR2},
+				{remoteVA, testPerspective3, testRIR3},
 			},
 		},
 		{
 			name:                     "functional localVA, 1 broken RVA, no CAA records",
 			domains:                  "present-dns-only.com",
 			localDNSClient:           caaMockDNS{},
-			expectedDiffLogSubstring: `RemoteSuccesses":2,"RemoteFailures":[{"VAHostname":"broken","Problem":{"type":"dns","detail":"While processing CAA for`,
+			expectedDiffLogSubstring: `RemoteSuccesses":2,"RemoteFailures":[{`,
 			remoteVAs: []RemoteVA{
-				{brokenVA, brokenUA},
-				{remoteVA, remoteUA},
-				{remoteVA, remoteUA},
+				{brokenVA, testPerspective1, testRIR1},
+				{remoteVA, testPerspective2, testRIR2},
+				{remoteVA, testPerspective3, testRIR3},
```

Comment:
All of the test cases here stop at 3 perspectives, but the number of allowed failures increases to 2 at 6+

Table: Quorum Requirements

| # of Distinct Remote Network Perspectives Used | # of Allowed non-Corroborations |
| --- | --- |
| 2-5 |  1  |
| 6+  |  2  |

We should add a test that checks this at exactly below and above that threshold.